### PR TITLE
docs: mock components with ng-mocks

### PR DESCRIPTION
**Pull request by maxime1992**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/spectator/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

No recommendation to mock components

Issue Number: N/A

## What is the new behavior?

Recommend to use [ng-mocks](https://github.com/ike18t/ng-mocks) 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,6 +270,26 @@ spectator.query(byLabel('By label'));
 spectator.query(byText('By text'));
 ```
 
+
+#### Mocking components
+
+If you need to mock components, you can use the [ng-mocks](https://github.com/ike18t/ng-mocks) library. Instead of using `CUSTOM_ELEMENTS_SCHEMA` which might hide some issues and won't help you to set inputs, outputs, etc, `ng-mocks` will auto mock the inputs, outputs, etc for you.
```
Comment (NetanelBasal) [inline README.md]: redudant space

File: README.md [hunk 2]
```diff
@@ -269,6 +270,26 @@ spectator.query(byLabel('By label'));
...
+
+Example:
+
+```ts
+import { createHostComponentFactory } from '@netbasal/spectator';
```
Comment (NetanelBasal) [inline README.md]: @ngneat

Comment (NetanelBasal) [reply README.md]: `createHostFactory`

File: README.md (hunk 1 above)
Comment (maxime1992) [reply README.md]: Not sure what you mean by that but I've removed the empty lines around the title

File: README.md (hunk 2 above)
Comment (maxime1992) [reply README.md]: Woups :zipper_mouth_face: took that from an old comment in the thread about ng-mocks :man_facepalming:

**Comment by NetanelBasal** [comment]
Thanks

**Comment by dirkluijk** [comment]
Awesome!

**Pull request merged**